pkgs/development/python-modules/skorch/default.nix

```diff
@@ -41,6 +59,9 @@ buildPythonPackage rec {
   ] ++ lib.optionals stdenv.isDarwin [
     # there is a problem with the compiler selection
     "test_fit_and_predict_with_compile"
+  ] ++ lib.optionals (python.pythonVersion == "3.11") [
+    # Python 3.11+ not yet supported for torch.compile
```

Comment:
is there a reference for this?

Reply:
This is the error returned when running the test. I added a reference to the relevant lines upstream.